src/components/spacecraft/ShipInfo.ts

```diff
@@ -0,0 +1,163 @@
+import axios from 'axios';
+import getLetterList from './SpeciesArrays';
+
+export interface StapiResponse{
```

Comment:
Place a space before the leading brace and check the rest of the braces in the file for same thing. Airbnb Style Guide 19.2